volmdlr/templates.py

```diff
@@ -108,15 +108,15 @@
         scene.lastEdgewidthUpdate = Date.now();
 
 
-        camera.onViewMatrixChangedObservable.add(() => {
-            if ((Date.now() - scene.lastEdgewidthUpdate) > 1000){
-                scene.lastEdgewidthUpdate = Date.now();
-                for (mesh of scene.meshes){
-                    var dist = BABYLON.Vector3.Distance(camera.position, mesh.position);
-                    mesh.edgesWidth = dist*0.1;
-                }
-            }
-         })
+        //camera.onViewMatrixChangedObservable.add(() => {
+        //    if ((Date.now() - scene.lastEdgewidthUpdate) > 1000){
+        //        scene.lastEdgewidthUpdate = Date.now();
+        //        for (mesh of scene.meshes){
+        //            var dist = BABYLON.Vector3.Distance(camera.position, mesh.position);
+        //            mesh.edgesWidth = dist*0.1;
+        //        }
+        //    }
+        // })
```

Comment:
Why disabling dynamic edge width?

Reply:
This is from the branch BSpline2D_fixes on which OpenTriangleShell3D is firstly created. Maybe I should change the destination branch from dev to BSpline2D_fixes ?